namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/messaging.tf

```diff
@@ -0,0 +1,128 @@
+module "offender_events" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sns-topic?ref=3.0"
+
+  team_name          = "${var.team_name}"
+  topic_display_name = "offender-events"
+
+  providers = {
+    aws = "aws.london"
+  }
+}
+
+module "offender_events_subscriber" {
```

Comment:
Can you please create all of the resources below in a separate PR after the module "offender_events"  is applied.

As other resources are depending upon that and pipeline is failing to apply the terraform with below error.
`  aws_iam_user.user: value of 'count' cannot be computed`

Reply:
will do

Reply:
ok @vijay-veeranki-moj done!